src/main/java/supercoder79/wavedefense/map/gen/WdPath.java

```diff
@@ -36,7 +34,7 @@ public static WdPath generate(int totalLength, int segmentLength) {
         int maxDeltaX = segmentLength / 2;
 
         while (currentLength < totalLength) {
-            int deltaX = MathHelper.nextInt(random, -maxDeltaX, maxDeltaX);
+            int deltaX = 0;//TODO random.next MathHelper.nextInt(random, -maxDeltaX, maxDeltaX);
```

Comment:
Should this still be a TODO? 😄

Reply:
Good catch! Fixed in 12e5bd72863335ff2445e4c700fc8f2183ef45cf